docs/guides/connecting_to_your_data/fluent/_fluent_datasources_assumed_code_for_guides.ipynb

```diff
@@ -0,0 +1,1365 @@
+    "  - This method utilizes `google.oauth2.service_account.Credentials.from_service_account_file` under the hood\n",
+    "- Passing a JSON string value to the optional `credentials` parameter when you create your GBQ Datasource\n",
+    "  - This string should contain the actual JSON data from your credentials file.\n",
+    "  - This method utilizes `google.oauth2.service_account.Credentials.from_service_account_info` under the hood"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d27e0bdb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Import GX and instantiate a Data Context\n",
+    "import great_expectations as gx\n",
+    "context = gx.get_context()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3dcb0716",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Create Datasource\n",
+    "datasource = context.datasources.add_spark_gbq(name=\"MyGbqDatasource\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "57c5c99d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Add GCS data to the Datasource as a Data Asset\n",
+    "csv_file_name = \"taxi_data.csv\"\n",
+    "data_asset = datasource.add_csv_asset(asset_name=\"MyTaxiDataAsset\", regex=csv_file_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0d907e6",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "\n",
+    "# SQL Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebec787f",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## How to connect to a SQL table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Prerequisites\n",
+    "- GX installed with SQL Dependencies\n",
+    "- Source data in a SQL Database\n",
+    "- Previously initialized Data Context\n",
+    "- SQL Credentials configured\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "657ae130",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Import GX and instantiate a Data Context\n",
+    "import great_expectations as gx\n",
+    "context = gx.get_context()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8c2de44c",
+   "metadata": {},
+   "source": [
+    "You can use either environment variables or a key in `config_variables.yml` to safely store any passwords needed by your connection string.  After defining your password in one of those ways, you can reference it in your connection string like this:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5c7bbfb1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Define your connection string\n",
+    "connection_string=\"postgresql+psycopg2://username:${MY_PASSWORD}@localhost/test\""
```

Comment:
You might want to use `postgres://` or `postgresql://` instead of `postgresql+psycopg2://` since they are more agnostic. The full list of postgres schemes we support is listed on the pydantic page:
https://docs.pydantic.dev/usage/types/#urls